pkg/v1/service/books/books.go

```diff
@@ -1,10 +1,36 @@
 package books
 
 import (
-    "context"
+	"context"
+	"log"
+	"strconv"
 )
 
-// GetBook returns the details of a given book ID. 
+// GetBook returns the details of a given book ID.
 func (s *bookSvc) GetBook(ctx context.Context, id string) (Book, error) {
-    return Book{Title: "Perdido Street Station"}, nil
+	rid, err := strconv.ParseUint(id, 10, 0)
+
+	if err != nil {
+		println("finding %v, %v", id, err)
```

Comment:
We can do this instead:
```suggestion
		fmt.Errorf("finding %v, %v", id, err)
```